### PR TITLE
AYON: Bug fix on Create Review in Maya

### PR DESCRIPTION
**Pull request by moonyuet**
## Changelog Description
This PR is to fix the error is found in Create Review due to the different compatibilities of the current playblast setting from the deprecated one.
## Additional info
["Viewport Options"] in preset will still support after the PR merged but will remove after users migrating all deprecated settings to the current ones

## Testing notes:
1. Build server addon with the code 
2. Install addon with the latest Openpype addon
3. Turn off Image Plane setting in playblast settings for the testing purpose
4. Launch Maya via launcher with the latest update of Openpype in AYON
5. Create Review
6. It is not errored out anymore

### CODE REVIEW

File: openpype/hosts/maya/plugins/create/create_review.py [hunk 1]
```diff
@@ -79,9 +79,16 @@ def create(self, subset_name, instance_data, pre_create_data):
...
-                "imagePlane": preset["Viewport Options"]["imagePlane"],
                 "panZoom": preset["Generic"]["pan_zoom"]
             }
+
+            if "Viewport Options" in preset:
+                mapping.update({
+                    "imagePlane": preset["Viewport Options"]["imagePlane"]})
+            elif "ViewportOptions" in preset:
+                mapping.update({
+                    "imagePlane": preset["ViewportOptions"]["imagePlane"]})
```
Comment (BigRoy) [inline openpype/hosts/maya/plugins/create/create_review.py:90]: Can we add a comment to state which is the deprecated one? Having a `# TODO: Deprecated after removal of X` type of comment can also help finding these spots that need fixing later.

Comment (moonyuet) [reply openpype/hosts/maya/plugins/create/create_review.py:90]: added

Comment (BigRoy) [reply openpype/hosts/maya/plugins/create/create_review.py:90]: I don't see it in the changes? :) Did you push it?

Comment (moonyuet) [reply openpype/hosts/maya/plugins/create/create_review.py:90]: It's my bad and we find out that it is not really related to creator settings but the conversion of the code to AYON settings.

Comment (BigRoy) [reply openpype/hosts/maya/plugins/create/create_review.py:90]: Great - so we can remove this logic / close the PR and fix it differently?

Comment (moonyuet) [reply openpype/hosts/maya/plugins/create/create_review.py:90]: > Great - so we can remove this logic / close the PR and fix it differently?

yes i will close it first and then make another PR for fix.(not a good branch name)

**Pull request closed without merging**